### PR TITLE
CKKS Bootstrapping

### PR DESCRIPTION
**Pull request by reneroliveira**
New Features:
- Enums for Security Level and Secret Key Distribution
- CKKS Bootstrapping (simple example)

### CODE REVIEW

**Comment by iquah1** [comment]
before we merge this lets first merge https://github.com/openfheorg/openfhe-python/pull/21

Happy to discuss more

File: src/bindings.cpp [hunk 1]
```diff
@@ -173,8 +191,23 @@ void bind_enums_and_constants(py::module &m)
         .value("INVALID_KS_TECH", KeySwitchTechnique::INVALID_KS_TECH)
         .value("BV", KeySwitchTechnique::BV)
         .value("HYBRID", KeySwitchTechnique::HYBRID);
+    // Secret Key Dist
+    py::enum_<SecretKeyDist>(m, "SecretKeyDist")
+        .value("GAUSSIAN", SecretKeyDist::GAUSSIAN)
+        .value("UNIFORM_TERNARY", SecretKeyDist::UNIFORM_TERNARY)
+        .value("SPARsE_TERNARY", SecretKeyDist::SPARSE_TERNARY);
```
Comment (iquah1) [inline src/bindings.cpp:198]: `SPARsE_TERNARY` -> `SPARSE_TERNARY`

**Review by iquah1 — CHANGES_REQUESTED** [review]
Small change

**Comment by reneroliveira** [comment]
> before we merge this lets first merge #21
> 
> Happy to discuss more

**Pull request merged**